### ISSUE TITLE
Move babel-polyfill to devDependencies

### ISSUE THREAD

**Issue by daedalus28** [smartprocure/futil-js#260]
Follow up to #182

We should evaluate if we even need it at all, but if we do, we should leave that up to consumers.

**Comment by suspiciousfellow** [comment]
Yes please. We have a ES5 environment running on Nashorn which spectacularly dies with babel polyfill.
